Intersect.Editor/Forms/DockingElements/frmMapLayers.Designer.cs

```diff
@@ -1377,6 +1375,11 @@ private void InitializeComponent()
 
         }
 
+        private void NudItemQuantity_ValueChanged(object sender, System.EventArgs e)
+        {
+            if (nudItemQuantity.Value < 1) nudItemQuantity.Value = 1;
+        }
+
```

Comment:
I don't think we want logic bundled into the designer code?

Reply:
Huh, interesting they ended up there.
That's definitely not intentional.

Changing it now.